hledger/Hledger/Cli/Commands/Register.hs

```diff
@@ -93,8 +93,9 @@ postingsReportItemAsCsvRecord (_, _, _, p, b) = [idx,date,code,desc,acct,amt,bal
                              BalancedVirtualPosting -> wrap "[" "]"
                              VirtualPosting -> wrap "(" ")"
                              _ -> id
-    amt = wbToText . showMixedAmountB oneLine $ pamount p
-    bal = wbToText $ showMixedAmountB oneLine b
+    -- We should already have stripped prices, so no need to do it again
```

Comment:
As elsewhere, it's a bit unclear for readers why/when prices were/should be stripped - perhaps you can add a little more context ?

Reply:
Got it.